databricks/koalas/tests/test_dataframe.py

```diff
@@ -553,6 +553,22 @@ def test_dot_in_column_name(self):
             ks.Series([1]),
         )
 
+    def test_droplevel(self):
```

Comment:
This method is introduced since pandas 0.24.0.

I think we can test this method only pandas > 0.24.0 like as below.

```python
def test_droplevel(self):
    # droplevel is new in pandas 0.24.0
    if LooseVersion(pd.__version__) >= LooseVersion("0.24.0"):
        pdf = (
            pd.DataFrame([[1, 2, 3, 4], [5, 6, 7, 8], [9, 10, 11, 12]])
            .set_index([0, 1])
            .rename_axis(["a", "b"])
        )

        pdf.columns = pd.MultiIndex.from_tuples(
            [("c", "e"), ("d", "f")], names=["level_1", "level_2"]
        )
        kdf = ks.from_pandas(pdf)
        self.assert_eq(pdf.droplevel("a"), kdf.droplevel("a"))
        self.assert_eq(pdf.droplevel("level_1", axis=1), kdf.droplevel("level_1", axis=1))
        self.assertRaises(ValueError, lambda: kdf.droplevel(["a", "b"]))
        self.assertRaises(ValueError, lambda: kdf.droplevel(["level_1", "level_2"], axis=1))
```